src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let StartIndex = 0;
```

Comment:
variable names should not be capitalized
```suggestion
    let startIndex = 0;
```